components/open-graph/SearchPreview.tsx

```diff
@@ -24,8 +24,8 @@ function SearchPreview({
   siteIcon,
   siteTimezone,
 }: SearchPreviewProps) {
-  const icon = useMedia(siteIcon);
-  const searchImage = useMedia(imageId);
+  const icon: any = useMedia(siteIcon);
+  const searchImage: any = useMedia(imageId);
```

Comment:
There is an interface added in #152 to support these types. We can add it in under that PR depending on where it lands.